apps/web/src/app/layout.tsx

```diff
@@ -47,12 +48,14 @@ export default function RootLayout({
   // If you want to develop locally with with Clerk,  Comment the provider below
   return (
     <ClerkProvider>
-      <html lang="en">
+      <html lang="en" suppressHydrationWarning>
```

Comment:
Next-themes Theme Provider component causes hydration mismatches. So one of the solutions is to add suppressHydrationWarning to the html element, as said in the [docs](https://nextjs.org/docs/messages/react-hydration-error).

We can avoid suppressing warnings by using solution number 2 from the [docs](https://nextjs.org/docs/messages/react-hydration-error#solution-2-disabling-ssr-on-specific-components). That means that we need to use dynamic import and disable prerendering.

```typescript
import dynamic from "next/dynamic";

const ThemeProvider = dynamic(() => import("../components/theme-provider"), {
  ssr: false,
});
```

I can make a change if you want to avoid suppressing warnings.

Reply:
Legend. Thanks for the explanation! Nothing to change here.